src/docker/docker-compose.yml

```diff
@@ -48,10 +48,13 @@ services:
       - MYSQL_DATABASE=tests
       - MYSQL_USER=prisma
       - MYSQL_PASSWORD=prisma
+    # https://stackoverflow.com/questions/55559386/how-to-fix-mbind-operation-not-permitted-in-mysql-error-log
+    cap_add:
+      - SYS_NICE # CAP_SYS_NICE
```

Comment:
Looks unrelated, first time I see that.
Did it happen on your Linux machine?

Reply:
Yes, happened on my Linux machine